app/script/entity/message/Asset.coffee

```diff
@@ -77,7 +77,9 @@ class z.entity.Asset
   @return [Boolean] Is asset of type video
   ###
   is_video: ->
-    if @type is z.assets.AssetType.FILE and @file_type?.startsWith('video') and not z.util.Environment.browser.firefox
+    is_video_asset = @type is z.assets.AssetType.FILE and @file_type?.startsWith 'video'
+    is_supported_browser = not (z.util.Environment.browser.firefox and platform.version < 49)
```

Comment:
I don't want to be an ass.. but I think the inequality change is _clearer_ than again using the not :)

```
platform.version > 48
```
or
```
platform.version >= 49
```